src/Components/Authorization/DesktopModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import Icon from "../Icon"
+import Modal, { ModalProps } from "../Modal/Modal"
+import Text from "../Text"
+import Colors from "../../Assets/Colors"
+
+const Header = styled.div`
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  padding: 10px;
+  margin: 20px 0 0;
+`
+
+const Content = styled.div`
+  box-sizing: border-box;
+  margin: 0px 60px;
+`
+
+const Logo = styled(Icon).attrs({
```

Comment:
Interesting, why do we do this? Wouldn't it be the same as:

```
const Logo = styled(Icon)`
  color: "black",
  font-size: "34px"
`
```

Reply:
Actually no it isn't the same, in this case it would have the same effect but this is a shorthand for passing props to the styled component, it also supports type definitions. For instance,

```typescript
import {IconProps, Icon} from './Icon'

interface LogoProps extends IconProps {
  secondaryColor: string
}

const Logo = styled(Icon).attrs<LogoProps>({
  secondaryColor: 'green'
})``;
```
this is equivalent to `<Logo secondaryColor="green" />` in this case